### PR TITLE
Update Attendance_Registry.py

### PR DESCRIPTION
**Pull request by maniuverse**

### CODE REVIEW

**Comment by maniuverse** [comment]
plz make change

File: UnsortedArray/Attendance_Registry.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 def Attendance_Registry(Entry):
-    Count=0
+    Count=1
```
Comment (Maria16pca111) [inline UnsortedArray/Attendance_Registry.py:2]: i know

**Review by Maria16pca111 — COMMENTED** [review]
Okay mb

**Comment by Maria16pca111** [comment]
Mani4

**Pull request merged**